README.md

```diff
@@ -1,10 +1,61 @@
-# "Finance and support for your business" application
+# Business Support Finder
 
-Frontend faceted search application for finding finance and support options.
+Faceted search application for finding business support and finance options.
 
-Built as a Rails frontend client to a separate API.
+## Screenshots
 
-## Requirements
+![Start Page](docs/assets/bsf-start.png)
 
-* [Business Support API](https://github.com/alphagov/business-support-api)
-* Ruby 2.1.5
+![Finder Results](docs/assets/bsf-finder.png)
+
+
+
+## Live examples
+
+- [gov.uk/business-finance-support-finder](https://www.gov.uk/business-finance-support-finder)
+- [gov.uk/business-finance-support-finder/search](https://www.gov.uk/business-finance-support-finder/search)
+
+
+## Technical documentation
+
+This is a Ruby on Rails application that is a frontend to [business-support-api](https://github.com/alphagov/business-support-api)
+
+It's a precursor to [finder-frontend](https://github.com/alphagov/finder-frontend) application, and should be merged in to it at some point. Currently it uses some filtering behaviour that `finder-frontend` does not support;
+
+- location/postcode filtering
```

Comment:
Do you think there might be a clearer way to describe this? I'm struggling to come up with one, but maybe we haven't quite identified what the technical difference is here?

Is it that `finder-frontend`'s filters map directly to a finder's facets while here we've got an extra step in the process (i.e. converting an entered postcode into a geographical area)?